test/spdm-responder-emu/src/main.rs

```diff
@@ -391,10 +391,7 @@ async fn handle_message(
         raw_packet.zeroize();
         let res = context.process_message(false, 0, raw_packet).await;
         match res {
-            Ok(spdm_result) => match spdm_result {
-                Ok(_) => continue,
-                Err(status) => panic!("process_message failed with {:?}", status),
-            },
+            Ok(_) => continue,
```

Comment:
I think we need give an example on how to process the Err(status), not just panic.